README.md

```diff
@@ -74,6 +74,20 @@ To run the tests, make sure to install the dependencies of the project using `np
 
 To run only behavior tests through jest-cucumber, you can use the command `npm run testBDD` which will run all the cucumber tests with corresponding `.feature` files.
 
+## Running the Mobile Application
+
+To run the app, you must have the expo cli installed on your machine. You also must set the Keycloak environment variable for authentication. 
+
+```
+npm install -g expo-cli // install the expo-cli if you haven't already
```

Comment:
nit: It might be better to add `expo-cli` as a dev dependency and execute through `npx` rather than requiring a global install, and this can be left as a note for convenience

Reply:
i think it talks about this in the expo docs so i left it there as an option.  does this look right so I can format it better? 6280e9d641f7cd05feb024e5907db355454c5970

Reply:
I think it would be nice to have `expo-cli` added to our `package.json` as a dev dependency so anyone (probably just me) can clone it and run `expo-cli` after installing the rest of the dependencies without needing to install `expo-cli`globally. This is probably just a me thing though

Reply:
readme change: 5551b286b8daacca1275a592c8ddefbfb747c508
addingDevDependency change: 5b1829df1768443f325aa93f17a8bf63ef444683